packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -184,12 +273,33 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
 <h1 className="sd-headline sd-headline--size-3xl bottom-4">Technical Questions</h1>
 <sd-accordion-group>
 <sd-accordion>
-  <h3 className="text-base sb-unstyled" slot="summary"> Why do we rely on basic HTML standards?</h3>
+  <h3 className="text-base sb-unstyled" slot="summary">Why do we rely on basic HTML standards?</h3>
   <span className="sb-unstyled">
     We rely on basic HTML standards because they provide a solid foundation for web development, ensuring compatibility across different browsers and devices. HTML standards define the structure and semantics of web content, which is crucial for accessibility and search engine optimization. By adhering to these standards, we promote consistency, reliability, and interoperability in web development, ultimately enhancing user experience and facilitating collaboration within the web community.
   </span>
 </sd-accordion>
 
+<sd-accordion>
+  <h3 className="text-base sb-unstyled" slot="summary">
+    Which browsers are supported?
+  </h3>
+  <span className="sb-unstyled">
+    Currently we provide automatic VRT in Chrome and manually test our components with MS Edge, Safari and Firefox. A
+    more detailed browser matrix is in progress and will be added here soon.
+  </span>
+</sd-accordion>
+
+<sd-accordion>
+  <h3 className="text-base sb-unstyled" slot="summary">
+    Does the design system support TypeScript and work with React, Vue, and Angular?
```

Comment:
Should we add Svelte? @karlbaumhauer

Reply:
ah yes. good point. 

@mariohamann is there anything we should better not mention as we already know about dificulties?

Reply:
@mariohamann could you answer this, pls?

Reply:
Yes: https://custom-elements-everywhere.com

Maybe we can add the link to this page?